lib/assets/javascripts/twine.js.coffee

```diff
@@ -274,7 +280,11 @@ Twine.bindingTypes =
     return refresh: ->
       newValue = fn.call(node, context, rootContext)
       for key, value of newValue when lastValue[key] != value
-        $(node).attr(key, value || null)
+        if value
+          value = value.call() if typeof value == 'function'
```

Comment:
Does this look okay? The jQuery `attr()` method allows for the second argument to be a function. This gets the test to pass but not sure if we should allow for the function to have arguments